### ISSUE TITLE
phpcbf fails with Foxland ruleset

### ISSUE THREAD

**Issue by nextgenthemes** [samikeijonen/phpcs-composer#1]
I am getting errors like this when the Foxland default ruleset is active. The absolute path gets appended to paths to the ruleset `...Foxland-Default/media/bla/web/app/plugin...` this should not happen.

Fixed it by disabling the Foxland ruleset.

```
./vendor/bin/phpcbf admin
.PHP Fatal error:  Uncaught PHP_CodeSniffer\Exceptions\RuntimeException: file_put_contents(/media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/samikeijonen/phpcs-composer/Foxland-Default/media/bla/web/app/plugins/advanced-responsive-video-embedder/admin/partials/debug-info.php): failed to open stream: No such file or directory in /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Reports/Cbf.php on line 90 in /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Runner.php:562
Stack trace:
#0 [internal function]: PHP_CodeSniffer\Runner->handleErrors(2, 'file_put_conten...', '/media/unencryp...', 90, Array)
#1 /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Reports/Cbf.php(90): file_put_content in /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Runner.php on line 562
.PHP Fatal error:  Uncaught PHP_CodeSniffer\Exceptions\RuntimeException: file_put_contents(/media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/samikeijonen/phpcs-composer/Foxland-Default/media/bla/web/app/plugins/advanced-responsive-video-embedder/admin/functions-admin.php): failed to open stream: No such file or directory in /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Reports/Cbf.php on line 90 in /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Runner.php:562
Stack trace:
#0 [internal function]: PHP_CodeSniffer\Runner->handleErrors(2, 'file_put_conten...', '/media/unencryp...', 90, Array)
#1 /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Reports/Cbf.php(90): file_put_contents('/ in /media/bla/web/app/plugins/advanced-responsive-video-embedder/vendor/squizlabs/php_codesniffer/src/Runner.php on line 562
. 3 / 3 (100%)
```

**Comment by samikeijonen** [comment]
Yeah I noticed the same issue. I'm not sure why it's getting absolute path in the first place, or what is the issue.

Let me know, if you know :)

**Comment by nextgenthemes** [comment]
Do you know why you need a `.` when calling phpcs and phpcbf. I came aware of your project from the mythic theme. If I remember correctly b4 the config was changed to this it was callable without any path. Your ruleset has the `<file>.</file>` declaration but you need to provide the file/path anyway. You know why? Putting it in the main config file works.

I do not know if there are other rules that do only work or are intended for the root config this `<arg name="basepath" value="./"/>` looks like it might be related to some path messup, not tested anything yet.

Honestly I am not sure if I like your small ruleset. Having those few rules and setting directly inside the mythic theme phpcs config file would be less complicated.

**Comment by samikeijonen** [comment]
> Honestly I am not sure if I like your small ruleset.

That's OK:)

**Comment by samikeijonen** [comment]
And thanks for the tip!!

I moved this to theme itself: `<arg name="basepath" value="./"/>`, and now it works.

**Comment by samikeijonen** [comment]
Fixed in c1d598acdea696e3b9e1a8687e22389b465f8c27 and 9fa0b24dc2636680acc2b73bfb22ef825a83e72e.
